src/signals/triage_signals.R

```diff
@@ -150,11 +156,27 @@ approve_signals <- function(df, fn_signals, test) {
     "DELETE: Delete the campaign content and `output/{indicator}/signals.parquet`",
     "file so you can recreate later.\n",
     "ARCHIVE: Delete the email campaign, but move the alert to `output/signals.parquet`\n",
-    "Any other input: Do nothing, so you can decide later.\n",
+    "DO_NOTHING: Do nothing, so you can decide later.\n",
     sep = ""
   )
 
-  user_command <- readline(prompt = "Your command: ")
+  user_command <- readline(prompt = "Your command: ")}
+  if (user_command %in% c("APPROVE", "DELETE")) {
+    # send message to user
+    cat(
+      "You typed the command: ", user_command, "\n",
+      "Given the criticality of the action please type I CONFIRM to proceed with the action selected",
+      sep = ""
+    )
+
+    user_command_confirmation <- readline(prompt = "Your command: ")
+    if (user_command_confirmation != "I CONFIRM"){
+      stop(
```

Comment:
Use `call. = FALSE` so that the error message appears cleaner to the user. Basically, it means, it won't show the traceback, which they don't necessarily need here, it's more about what they type as input.